programs/fileio.c

```diff
@@ -624,7 +630,7 @@ FIO_openDstFile(FIO_prefs_t* const prefs,
  * @return : loaded size
  *  if fileName==NULL, returns 0 and a NULL pointer
  */
-static size_t FIO_createDictBuffer(void** bufferPtr, const char* fileName)
+static size_t FIO_createDictBuffer(FIO_prefs_t* const prefs, void** bufferPtr, const char* fileName)
```

Comment:
minor : API : argument order

Prefer grouping the _mutable states_ at the beginning.

In this case, `bufferPtr` is the actual of this function (it will be modified), while `prefs` is merely a parameter (it will only be read).